src/main/java/soju/commands/ByeCommand.java

```diff
@@ -0,0 +1,18 @@
+package soju.commands;
+
+import soju.Storage;
+import soju.TaskList;
+import soju.Ui;
+
+/**
+ * ByeCommand handles commands starting with bye
+ */
+public class ByeCommand extends Command {
+    public ByeCommand(String input) {
+    }
```

Comment:
Your ByeCommand constructor seems to not do anything, is there a reason why it was left empty?